apps/ewallet/lib/ewallet/gates/transactions/childchain_gate.ex

```diff
@@ -31,6 +31,7 @@ defmodule EWallet.TransactionGate.Childchain do
   end
 
   defp build_transaction_attrs(%{"address" => address} = attrs) do
+    # this doesn't exist anymore with ALD and need to be replaced with call to the vaults
```

Comment:
can you replace it in the PR or is this a larger task?

Reply:
it's big :)
It'll be in this PR: https://github.com/omgnetwork/ewallet/pull/1217/files